e2e/suites/share-entities.test.ts

```diff
@@ -28,7 +28,7 @@ describe('Share entities', () => {
   const getEntitiesCollaborators = async () =>
     page.$$eval('.members-list tr .member-list-item', items => items.map(item => item.textContent));
 
-  const checkAccessOfPersons = (accesses: string[]) => {
+  const checkAccessOfPersons = async (accesses: string[]) => {
```

Comment:
@LaszloKecskes I think for this function to be properly awaitable it needs to return a Promise.all of all the resulting promises from the map()